vro-types/o11n-plugin-aria/index.d.ts

```diff
@@ -10,16 +10,429 @@ declare class VraHost {
+	createdAt: string;
+	folder: string;
+	tagsToMatch: Object[];
+	internalIdString: string;
+	name: string;
+	host: VraHost;
+	placementPolicy: string;
+	tagsExtension: string;
+	tagsToMatchExtension: string;
+	id: string;
+	updatedAt: string;
+	customPropertiesExtension: string;
+	constructor();
+	constructor();
+
+	/**
+	 * @param tagsToMatchItem
+	 */
+	addTagsToMatchItem(tagsToMatchItem: VraTag): VraZone;
+
+	/**
+	 * @param key
+	 * @param linksItem
+	 */
+	putLinksItem(key: string, linksItem: VraHref): VraZone;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraZone;
+
+	/**
+	 * @param key
+	 * @param customPropertiesItem
+	 */
+	putCustomPropertiesItem(key: string, customPropertiesItem: string): VraZone;
+}
+
+declare class VraHref {
+	hrefs: Object[];
+	href: string;
+	constructor();
+	constructor();
+
+	/**
+	 * @param hrefsItem
+	 */
+	addHrefsItem(hrefsItem: string): VraHref;
+}
+
+/**
+ * Specification for a zone.
+ */
+declare class VraZoneSpecification {
+	folder: string;
+	computeIds: Object[];
+	tagsToMatch: Object[];
+	regionId: string;
+	name: string;
+	placementPolicy: string;
+	description: string;
+	tags: Object[];
+	constructor();
+	constructor();
+
+	/**
+	 * @param tagsToMatchItem
+	 */
+	addTagsToMatchItem(tagsToMatchItem: VraTag): VraZoneSpecification;
+
+	/**
+	 * @param key
+	 * @param customPropertiesItem
+	 */
+	putCustomPropertiesItem(
+		key: string,
+		customPropertiesItem: string
+	): VraZoneSpecification;
 
+	/**
+	 * @param computeIdsItem
+	 */
+	addComputeIdsItem(computeIdsItem: string): VraZoneSpecification;
+
+	/**
+	 * @param tagsItem
+	 */
+	addTagsItem(tagsItem: VraTag): VraZoneSpecification;
+}
+
+declare class VraTag {
+	readonly displayName: string;
+	readonly internalIdString: string;
+	host: VraHost;
+	value: string;
+	key: string;
+	constructor();
+	constructor();
+}
+
+declare class VraDataCollectorService {
+	constructor();
+
+	/**
+	 * @param genericRestClient
+	 */
+	setGenericRestClient(genericRestClient: VraGenericRestClient): void;
+}
+
+declare class VraRequestService {
+	constructor();
+
+	/**
+	 * @param genericRestClient
+	 */
+	setGenericRestClient(genericRestClient: VraGenericRestClient): void;
+
+	/**
+	 * Awaits successfully completed RequestTracker Response.
+	 * @param requestTracker
+	 */
+	awaitRequestTrackerResponse(
+		requestTracker: VraRequestTracker
+	): VraRequestTracker;
+
+	/**
+	 * Delete a request tracker object for a given request id.
+	 * @param requestTrackerId
+	 */
+	deleteRequestTracker(requestTrackerId: string): void;
+}
+
+declare class VraRequestTracker {
+	deploymentId: string;
+	readonly internalIdString: string;
+	name: string;
+	host: VraHost;
+	resources: Object[];
+	progress: number;
+	id: string;
+	requestTrackerStatus: string;
+	message: string;
+	selfLink: string;
+	constructor();
+
+	/**
+	 * An object used to track long-running operations.
+	 */
+	constructor();
+
+	/**
+	 * An object used to track long-running operations.
```

Comment:
Incorrect method description - remove

Reply:
Following the vRO API, it is a proper method description

Reply:
This description is for the constructor, not the **addResourcesItem** method